funsor/inverse_laplace.py

```diff
@@ -0,0 +1,80 @@
+# Copyright Contributors to the Pyro project.
+# SPDX-License-Identifier: Apache-2.0
+
+from math import pi
+
+from funsor.factory import Bound, Fresh, Has, make_funsor
+from funsor.interpretations import StatefulInterpretation
+from funsor.tensor import Tensor
+from funsor.terms import Funsor, Variable
+from funsor.util import get_backend
+
+from . import ops
+
+
+@make_funsor
+def InverseLaplace(
+    F: Has[{"s"}], t: Funsor, s: Bound  # noqa: F821
+) -> Fresh[lambda F: F]:
+    """
+    Inverse Laplace transform of function F.
```

Comment:
Say explicitly that there is no closed-form solution for arbitrary `F`, so we must resort to numerical approximations which we store in a new interpretation `Talbot`.